pkg/frontend/frontend_test.go

```diff
@@ -111,7 +111,8 @@ func TestFrontend_LogsSlowQueriesFormValues(t *testing.T) {
 
 	// Configure the query-frontend with the mocked downstream server.
 	config := defaultFrontendConfig()
-	config.Handler.LogQueriesLongerThan = 1 * time.Microsecond
+	// Setting to < 0 to ensure all queries are logged.
+	config.Handler.LogQueriesLongerThan = -1 * time.Microsecond
```

Comment:
I was a bit surprised that this is changing what the test does, rather than testing the old and new behaviour.
However I think the test should have had both a slow and a fast query to test the old one properly.

Reply:
I thought the reason this was set so low was to in fact log every query for the purpose of this test. But I agree to have both cases tested separately would be nicer, I'll add it.

Reply:
I added the test case, let me know what you think!